packages/inference/test/HfInference.spec.ts

```diff
@@ -829,6 +829,16 @@ describe.concurrent("HfInference", () => {
 				});
 				expect(res).toBeInstanceOf(Blob);
 			});
+
+			it("textToSpeech OuteTTS", async () => {
+				const res = await client.textToSpeech({
+					model: "OuteAI/OuteTTS-0.3-1B",
```

Comment:
(see my previous comment) do you want to keep the `1B` example and version or the `500M`?

```suggestion
					model: "OuteAI/OuteTTS-0.3-500M",
```

Reply:
oops no will fix

Reply:
28dd0a29509de2beb1de203704e8024c3270d3e6